src/library/kotlin/org/jetbrains/kotlin/konan/library/KonanLibrary.kt

```diff
@@ -63,11 +63,11 @@ val KonanLibrary.packageFqName
     get() = manifestProperties.getProperty(KLIB_PROPERTY_PACKAGE)
 
 val KonanLibrary.exportForwardDeclarations
-    get() = manifestProperties.getProperty(KLIB_PROPERTY_EXPORT_FORWARD_DECLARATIONS)
-            .split(' ').asSequence()
+    get() = manifestProperties.propertyList(KLIB_PROPERTY_EXPORT_FORWARD_DECLARATIONS, escapeInQuotes = true)
```

Comment:
Not sure if it makes any sense.

Reply:
I think it's better to do everything in one way, because during reading def files we don't choose for which fields it is used.